### PR TITLE
Updated Upload ReadMe for more instructions

### PR DESCRIPTION
**Pull request by pandermatt**
Updated ReadMe and made more usability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,16 @@ This is a component for the renuo upload (<https://github.com/renuo/renuo-upload
 
 ## How to use
 
+### Upload Components
+
+#### Single Upload Component
```
Comment (pandermatt) [inline README.md]: Link, as soon as merged

File: README.md [hunk 2]
```diff
@@ -15,6 +15,16 @@ This is a component for the renuo upload (<https://github.com/renuo/renuo-upload
...
+
+![simple-upload](https://cloud.githubusercontent.com/assets/20790833/26151729/54d950c2-3b04-11e7-83b0-cee18cb44a32.gif)
+
+#### Multi Upload Component
```
Comment (pandermatt) [inline README.md]: Link, as soon as merged

File: README.md [hunk 3]
```diff
@@ -15,33 +15,19 @@ This is a component for the renuo upload (<https://github.com/renuo/renuo-upload
...
 
-### Single Image
+### Upload Components
 
-#### Simple usage
+#### Single Upload Component
 
-```angular2html
-<ru-img [src]="picture"></ru-img>
-```
```
Comment (lukaselmer) [inline README.md:24]: i think it would be nice to have a link to the details here :)

File: documentation/gallery.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Simple Usage
```
Comment (lukaselmer) [inline documentation/gallery.md]: IMO the title should be `Gallery`, and `Simple Usage` would be the subtitle?

**Review by lukaselmer — APPROVED** [review]
Nice, LGTM 👍 the gifs are *very* nice 😄

File: documentation/gallery.md (hunk 4 above)
Comment (pandermatt) [reply documentation/gallery.md]: Yes, thank you

**Pull request merged**